recipes/nodejs/all/conanfile.py

```diff
@@ -29,20 +37,37 @@ def _nodejs_arch(self):
         return str(self.settings.arch)
 
     def validate(self):
-        if (not (self.version in self.conan_data["sources"]) or
-            not (str(self.settings.os) in self.conan_data["sources"][self.version]) or
-            not (self._nodejs_arch in self.conan_data["sources"][self.version][str(self.settings.os)] ) ):
+        if not self.version in self.conan_data["sources"] or \
+           not str(self.settings.os) in self.conan_data["sources"][self.version] or \
+           not self._nodejs_arch in self.conan_data["sources"][self.version][str(self.settings.os)]:
             raise ConanInvalidConfiguration("Binaries for this combination of architecture/version/os not available")
 
+        building_doc_url = "https://github.com/nodejs/node/blob/v{}/BUILDING.md".format(str(self.version))
```

Comment:
Why do you check here the build-from-source requirements when the recipe still uses the pre-compiled binaries?